### DISCUSSION TITLE
How to recovered a forgotten passcode in ipad

### DISCUSSION THREAD

**Discussion by puentejo** [community/community#139560] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Good evening. I forgot my passcode for my ipad
Is there a way to reset the ipad passcode?
Thank you

**Comment by ebndev** [comment]
Hi @puentejo,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this, you might want to try asking for help somewhere that focuses on this project, such as [Apple's Support Forum](https://discussions.apple.com/welcome). 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
